wp-content/themes/pub/wporg-learn-2020/css/components/_card.scss

```
@@ -128,5 +128,18 @@
 				flex-basis: 48%;
 			}
 		}
+
+		&_4 {
+			> * {
+				flex-grow: 0;
+				flex-shrink: 0;
+				flex-basis: calc(25% - 16px);
```

Maybe best to follow the other units in this module?
```suggestion
				flex-basis: calc(25% - 1rem);
```